### PR TITLE
fix: Flow layout issues

### PR DESCRIPTION
**Pull request by DafyddLlyr**
## What does this PR do?
Resolves layout issues described in https://trello.com/c/pKlx6bg5/2979-diagnose-and-fix-layout-overlap-in-editor-graph

## Testing
Please compare the following two flows - 
 - https://editor.planx.dev/open-digital-planning/validating-and-assessing-applications-with-heritage-impacts
 - https://3499.planx.pizza/open-digital-planning/validating-and-assessing-applications-with-heritage-impacts

### CODE REVIEW

**Comment by DafyddLlyr** [comment]
[Diagnose and fix layout overlap in editor graph](https://trello.com/c/pKlx6bg5/2979-diagnose-and-fix-layout-overlap-in-editor-graph)

File: editor.planx.uk/src/pages/FlowEditor/floweditor.scss [hunk 1]
```diff
@@ -47,7 +47,8 @@ $fontMonospace: "Source Code Pro", monospace;
   li {
     list-style: none;
     padding: 0;
-    margin: 0;
+    margin: 0 auto;
+    width: max-content;
```
Comment (ianjon3s) [inline editor.planx.uk/src/pages/FlowEditor/floweditor.scss:51]: This is fixing the overlap but reducing 100% width of section components (needed for dashed divider line). There is an appended class for section list items (`type-Section`) so adding 100% width for that class should fix it.

**Review by ianjon3s — APPROVED** [review]
After further flow testing this has resolved layout issues and introduce not apparent regressions.

**Comment by ianjon3s** [comment]
Merging this in @DafyddLlyr 's absense.

**Pull request merged**